core/src/main/java/com/graphhopper/routing/util/parsers/OSMTurnRelationParser.java

```diff
@@ -103,7 +103,7 @@ private EdgeExplorer getOutExplorer(Graph graph) {
     /**
      * Add the specified relation to the TurnCostStorage
      */
-    void addRelationToTCStorage(OSMTurnRelation osmTurnRelation, IntsRef turnCostFlags,
+    void addRelationToTCStorage(OSMTurnRelation osmTurnRelation,
```

Comment:
Interface is generic, but `OSMTurnRelation` is used... Maybe redesign?
Also, turn relations can have multiple edges (which is already wrong in GH [should be](https://www.openstreetmap.org/relation/6655738) but [is](https://graphhopper.com/maps/?point=52.402065%2C16.921917&point=52.402046%2C16.922046&locale=en-US&vehicle=car&weighting=fastest&turn_costs=true&use_miles=false&layer=Omniscale)), but there... abstraction leaks even more.